supervisor/shared/usb/usb_msc_flash.c

```diff
@@ -135,9 +200,6 @@ int32_t tud_msc_write10_cb (uint8_t lun, uint32_t lba, uint32_t offset, void* bu
 void tud_msc_write10_complete_cb (uint8_t lun) {
     (void) lun;
 
-    // flush pending cache when write10 is complete
-    internal_flash_flush();
-
     // This write is complete, start the autoreload clock.
```

Comment:
How come you don't `internal_flash_flush()` any more?

Reply:
The write from the computer could have been a single sector and we should wait to flush until a write is outside the erase block. Thach added this in his implementation but I don't think its needed.